src/01-functions.js

```diff
@@ -39,7 +45,10 @@ function greet() {}
  *  add(10, 20);
  *  //> 30
  */
-function add() {}
+function add(a, b) {
+  resultAdd = a + b;
+  return resultAdd;
```

Comment:
This function is great - its clean and very easy for any developer to follow.  It is possible to simply return `a + b` without saving to variable if you wanted to make this one line of code.